config.go

```diff
@@ -0,0 +1,10 @@
+package plugin
+
+// Configor is the interface plugins should implement in order to provide configuration interface to the user
+type Configor interface {
```

Comment:
The Configurer needs a `Schema() string` method, to provide the schema of the configuration

Reply:
Is it possible to just use the default config as the schema?

Reply:
We can generate a schema from the struct see https://github.com/alecthomas/jsonschema but I would prefer to use EmptyConfig if https://github.com/gotify/plugin-api/pull/1#discussion_r247315460 will be done.